testing/wcag/.pa11yci.ci.json

```diff
@@ -0,0 +1 @@
+{"urls":["http://host.docker.internal:6006/iframe.html?id=2-design-foundations--colour-language","http://host.docker.internal:6006/iframe.html?id=2-design-foundations--colour-palette","http://host.docker.internal:6006/iframe.html?id=3-components--advice-feedback","http://host.docker.internal:6006/iframe.html?id=3-components--advice-feedback","http://host.docker.internal:6006/iframe.html?id=3-components--advice-feedback","http://host.docker.internal:6006/iframe.html?id=3-components--asset-hyperlink","http://host.docker.internal:6006/iframe.html?id=3-components--breadcrumb","http://host.docker.internal:6006/iframe.html?id=3-components--buttons","http://host.docker.internal:6006/iframe.html?id=3-components--callout","http://host.docker.internal:6006/iframe.html?id=3-components--contact-details","http://host.docker.internal:6006/iframe.html?id=3-components--error-summary","http://host.docker.internal:6006/iframe.html?id=3-components--footer","http://host.docker.internal:6006/iframe.html?id=3-components--header","http://host.docker.internal:6006/iframe.html?id=3-components--input","http://host.docker.internal:6006/iframe.html?id=3-components--logo","http://host.docker.internal:6006/iframe.html?id=3-components--navigation","http://host.docker.internal:6006/iframe.html?id=3-components--notice-banner","http://host.docker.internal:6006/iframe.html?id=3-components--page-review","http://host.docker.internal:6006/iframe.html?id=3-components--pagination","http://host.docker.internal:6006/iframe.html?id=3-components--paging-info","http://host.docker.internal:6006/iframe.html?id=3-components--radio-group","http://host.docker.internal:6006/iframe.html?id=3-components--radio-group-small","http://host.docker.internal:6006/iframe.html?id=3-components--search","http://host.docker.internal:6006/iframe.html?id=3-components--success-message","http://host.docker.internal:6006/iframe.html?id=3-components--targeted-content","http://host.docker.internal:6006/iframe.html?id=3-components--textarea","http://host.docker.internal:6006/iframe.html?id=3-components--website-feedback","http://host.docker.internal:6006/iframe.html?id=3-components--related-content","http://host.docker.internal:6006/iframe.html?id=3-components--tables"]}
```

Comment:
can we make these multi-lines? Or do they need to be one line?

Reply:
it's automatically generated by the task, so it doesn't really matter.